gateway/src/apicast/policy/soap/soap.lua

```diff
@@ -43,14 +44,25 @@ end
 -- Content-Type header. The string has this format:
 -- a_param=x;action=soap_action;another_param=y
 -- This method returns the value of 'action' or nil when it's not present.
+-- Note that params can be a mix of lower and upper case chars, it can have
+-- spaces, and wrap the value with '"':
+-- https://tools.ietf.org/html/rfc7231#section-3.1.1.1
 local function soap_action_from_ctype_params(params)
-  local params_split = re.split(params, ";")
+  local params_without_blanks = ngx.re.gsub(params, '\\s', '')
+  local params_split = re.split(params_without_blanks, ";")
```

Comment:
You could also split by `[[\s*;\s*]]` to remove spaces around `;`, right?
Seems a bit evil to strip all spaces from it. What if there is a space in the url? Now when I think about it. 
What happens when there is `;` in soap action? That would be quite broken.